Project.toml

```diff
@@ -37,4 +41,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
 docs = ["Documenter"]
-test = ["DataFrames", "Test", "MLJBase", "MLJTestInterface"]
+test = ["CUDA", "DataFrames", "Test", "MLJBase", "MLJTestInterface"]
+
+[weakdeps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
From Julia's formatter, it appears that expected order of project keys to be:
- deps
- compat
- weakdeps
- extensions
- compat
- extras
- targets

Reply:
It depends on the Julia version you use: On Julia 1.6, the order Pkg uses is
- deps
- compat
- extensions
- extras
- targets
- weakdeps
whereas on Julia 1.7, 1.8, and 1.9 it is
- deps
- weakdeps
- extensions
- compat
- extras
- targets

See eg the discussion in https://github.com/JuliaTesting/Aqua.jl/issues/105 and in particular https://github.com/JuliaTesting/Aqua.jl/issues/105#issuecomment-1551221369

I guess you were referring to the order in Julia > 1.6 and accidentally included the compat section twice?

Reply:
Yes I meant to refer to Julia > 1.6, sorry about the confusion for the duplicated `compat`.

Reply:
I updated the order to the convention in Julia > 1.6.